docs/user_manual/source/control_status_registers.rst

```diff
@@ -1632,11 +1632,135 @@ Detailed:
   +-------+------+------------------------------------------------------------------+
   | Bit#  | R/W  | Description                                                      |
   +=======+======+==================================================================+
-  | 31:0  | RW   | **DATA**                                                         |
+  | 31:0  | WARL | **DATA**                                                         |
   +-------+------+------------------------------------------------------------------+
 
-Accessible in Debug Mode or M-Mode, depending on **tdata1.dmode**.
-This register stores the instruction address, load address or store address to match against for a breakpoint trigger or the currently selected exception codes for an exception trigger.
+.. note::
+   The WARL behavior of ``tdata2`` depends on the values of ``tdata1.TYPE`` and ``tdata1.DMODE`` as described in
+   :ref:`csr-tdata2-type-0x2`, :ref:`csr-tdata2-type-0x5`, :ref:`csr-tdata2-type-0x6` and :ref:`csr-tdata2-type-0xf`.
+
+.. _csr-tdata2-type-0x2:
+
+Trigger Data Register 2 (``tdata2``) - View when ``tdata1.TYPE`` is 0x2
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+CSR Address: 0x7A2
+
+Reset Value: 0x0000_0000
+
+Detailed:
+
+.. table::
+  :widths: 10 20 70
+  :class: no-scrollbar-table
+
+  +-------+------+------------------------------------------------------------------+
+  | Bit#  | R/W  | Description                                                      |
+  +=======+======+==================================================================+
+  | 31:0  | WARL | **DATA**                                                         |
+  +-------+------+------------------------------------------------------------------+
+
+.. note::
+   Accessible in Debug Mode or M-Mode, depending on ``tdata1.DMODE``.
+   This register stores the instruction address, load address or store address to match against for a breakpoint trigger.
+
+.. _csr-tdata2-type-0x5:
+
+Trigger Data Register 2 (``tdata2``) - View when ``tdata1.TYPE`` is 0x5
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+CSR Address: 0x7A2
+
+Reset Value: 0x0000_0000
+
+Detailed:
+
+.. table::
+  :widths: 10 20 70
+  :class: no-scrollbar-table
+
+  +-------+------------+------------------------------------------------------------------+
+  | Bit#  | R/W        | Description                                                      |
+  +=======+============+==================================================================+
+  | 31:25 | WARL (0x0) | **DATA[31:25]**                                                  |
+  +-------+------------+------------------------------------------------------------------+
```

Comment:
Add description for bit 25: WARL **DATA[25]**. Instruction parity/checksum fault.

Reply:
Added description for bits 8 (ecall from u-mode) and 25 as mentioned.